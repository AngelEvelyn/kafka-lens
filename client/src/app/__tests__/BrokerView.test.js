// create test for BrokerView
// bring in fake state
// test snap shot
// test if active it should be green, if not it should be red
// should render Broker Header and BrokerTopicsView
