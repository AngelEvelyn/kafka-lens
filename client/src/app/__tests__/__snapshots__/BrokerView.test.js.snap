// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`BrokerView.js unit tests Should render without errors 1`] = `
<div
  className="brokerView"
  style={
    Object {
      "backgroundColor": "#E9573F",
    }
  }
>
  <BrokerHeader />
  <BrokerTopicsView />
</div>
`;
